.github/workflows/ci.yml

```
@@ -3,49 +3,41 @@ name: CI
 on:
   push:
     branches:
-      - master
+    - master
   pull_request:
     branches:
-      - master
+    - master
 
 jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        vim_version:
-        - 'v8.2.2000'
-        - 'v8.2.1000'
-        - 'v8.2.0000'
-        - 'v8.1.0000'
-        - 'v8.0.0000'
-        - 'v7.4'
+        vim:
+        - v8.2.1000
+        - v8.2.0000
+        - v8.1.0000
+        - v8.0.0000
+        - v7.4
 
     steps:
     - name: Checkout code
-      uses: actions/checkout@master
+      uses: actions/checkout@v2
```

v3 has been released.

```suggestion
      uses: actions/checkout@v3
```

---

@uhooi 
Thanks, I accept this patch.